cmd/cmd/carbonAppInfo.go

```diff
@@ -80,9 +80,11 @@ func printAppHelp() {
 
 func executeGetCarbonAppCmd(appname string) {
 
-	finalUrl := utils.RESTAPIBase + utils.PrefixCarbonApps + "?carbonAppName=" + appname
+	finalUrl := utils.RESTAPIBase + utils.PrefixCarbonApps
+	params := make(map[string]string)
+	params["carbonAppName"] = appname
```

Comment:
I see these three lines are repeated in few places. Only difference is the keyname and value. Can we refactor code to avoid duplicates?